src/main/AntidoteBackend.java

```diff
@@ -0,0 +1,114 @@
+package main;
+
+import com.ericsson.otp.erlang.OtpErlangBinary;
+import com.ericsson.otp.erlang.OtpErlangAtom;
+import com.ericsson.otp.erlang.OtpErlangPid;
+import com.ericsson.otp.erlang.OtpErlangTuple;
+import com.ericsson.otp.erlang.OtpMbox;
+import com.ericsson.otp.erlang.OtpNode;
+
+abstract class AntidoteBackend {
+    OtpMbox myOtpMbox = null;
+    OtpNode myOtpNode = null;
```

Comment:
maybe `final OptMbox myOptMbox;` (and same for node)

Reply:
resolved